src/js/pages/OffersList/components/SorterFilter.jsx

```diff
@@ -64,9 +64,9 @@ class FilterMenu extends Component {
             <h5 className="mt-4">Payment method</h5>
             <span className="text-muted text-small">Popular</span>
             <ButtonGroup vertical className="w-100">
-              {POPULAR_PAYMENT_METHODS.map((index) => (
+              {POPULAR_PAYMENT_METHODS_INDEXES.map((index) => (
                 <CheckButton active={index === props.paymentMethodFilter}
-                            key={'paymentMethod-' + index} isCheckBox
```

Comment:
Why remove isCheckBox?

Reply:
they were originally radio buttons, and due to copy-paste I turned them into a checkbox. With this, I'm returning it back to their correct format.

Reply:
Well, they should be check boxes though. Check the design and also the code, we can select multiple ones

Reply:
Even in the selection filter for the offer list?

Reply:
Oh... 🤦‍♂ 
My bad I hadn't realized this was the filter. The filter should be radio, but the seller page checkbox